projects/laji/src/app/+project-form/project-form.component.ts

```diff
@@ -131,21 +136,21 @@ export class ProjectFormComponent implements OnInit, OnDestroy {
       startWith(this.router.url)
     );
 
-    this.showNav$ = routerEvents$.pipe(
-      mergeMap((url: string) =>
-        form$.pipe(
-          map(form =>
-            !(
-              (!form.options?.useNamedPlaces && url.match(/\/form$/))
-              || (form.options?.useNamedPlaces && url.match(/\/places\/MNP\.\d+$/))
-              || (url.match(/\/form\/(.*\/)?((JX\.)|(T:))\d+$/))
-              || (form.options?.useNamedPlaces && (url.match(/\/form\/places/) || url.match(/\/form\/MHL.*\/places/)))
-            )
+    this.showNav$ = combineLatest(this.browserService.lgScreen$, routerEvents$, this.isNavabarToggled$).pipe(
+      mergeMap(([isDesktopScreen, url, isNavbarToggled]) => 
+      form$.pipe(
+        map(form =>
+          !(
+            (!form.options?.useNamedPlaces && url.match(/\/form$/))
+            || (form.options?.useNamedPlaces && url.match(/\/places\/MNP\.\d+$/))
+            || (url.match(/\/form\/(.*\/)?((JX\.)|(T:))\d+$/))
+            || (!isDesktopScreen && !isNavbarToggled)
           )
         )
       )
-    );
-
+      )
+    )
+    
```

Comment:
Looks good! Could you run the indentation formatting tool of your IDE on this block? The indentation at line 141 should be one step deeper.